features/admin.applications.v1/components/wizard/minimal-application-create-wizard.tsx

```
@@ -1248,6 +1287,92 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
+                                when={ featureConfig?.applications?.scopes?.update }
+                            >
+                                <Grid.Row columns={ 1 }>
+                                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 14 }>
+                                        <Field
+                                            name="applicationTokenIssuer"
+                                            label={ t("applications:forms.inboundOIDC" +
+                                                ".sections.applicationTokenIssuer.fields.tokenIssuer.label") }
+                                            type="dropdown"
+                                            required={ false }
+                                            value={ selectedTokenEndpoint }
+                                            children={
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)